packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,282 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Tooltip Component"}}
+
+<Shw::Text::H1>Tooltip</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>TooltipButton</Shw::Text::H2>
+
+  <Shw::Text::H3>Content</Shw::Text::H3>
```

Comment:
I think we're missing a use case with `allowHTML=true` (and the tooltip already opened, so it can be covered by visual regression testing)